standard/currency/currency.lua

```diff
@@ -92,4 +92,19 @@ function Currency.formatMoney(value)
 	return LANG:formatNum(integer) .. string.format('%.2f', decimal):sub(2)
 end
 
+function Currency.getExchangeRate(props)
+	local setVariables = Logic.readBool(props.setVariables)
+	local currencyRate = tonumber(props.currencyRate)
+	local currency = props.currency:upper()
+	if not currencyRate then
+		currencyRate = mw.ext.CurrencyExchange.currencyexchange(1, currency, USD:upper(), props.date)
+	end
+
+	if setVariables and currencyRate and String.isNotEmpty(currencyRate) then
+		Variables.varDefine(currency .. '_rate', currencyRate)
+	end
```

Comment:
Looks like this is being defined in two places.

Reply:
kicked one